us_to_metric.md

```diff
@@ -0,0 +1,33 @@
+
+# A note on US measurements
+
+1 tablespoon = 15g / 15 mL*
+1 teaspoon = 5g / 5ml*
+1 fluid ounce (1/8 cup) = 30mL*
+1/4 cup (2 fluid ounces) = 60mL*
+1/2 cup = 120mL*
+1 cup = 240mL*
+1 pint = 2 cups 470mL*
+1 quart = 950 mL*
+1 gallon = 3.8L*
+1 ounce = 30g*
+1 pound = 450g*
+
+1 cup of flour = 140g*
+1 cup of white/brown sugar = 200g*
+1 cup of powdered sugar = 120g*
+1 cup cocoa powder = 85g*
+
+1 tablespoon butter = 1/2 ounce / 15g*
+1 stick butter = 8 tablespoons = 1/2 cup = 4 ounces / 110g*
+2 sticks butter = 1 cup = 8 ounces / 225g*
+
+* approximately
+
+##Oven Temperature
+
+cool       200F = 90C
+slow       300F = 150C
+moderate   350F = 180C
+hot        400F = 200C
+fast       450F = 230C
```

Comment:
```suggestion
# [US measurements](https://en.wikipedia.org/wiki/United_States_customary_units)

_(All measures are approximate)_

US            |                | Metric
--            | --             | --:
1 tablespoon  | 1/2 fluid ounce| 15g / 15 mL
1 teaspoon    |                | 5g / 5ml
1 fluid ounce | 1/8 cup        | 30mL
1/4 cup       | 2 fluid ounces | 60mL
1/2 cup       | 4 fluid ounces | 120mL
1 cup         | 8 fluid ounces | 240mL
1 pint        | 2 cups         | 470mL
1 quart       |                | 950 mL
1 gallon      |                | 3.8L
1 ounce       |                | 30g
1 pound       | 16 ounces      | 450g

## Ingredient density

Ingredient         | Mass of 1 cup | Mass of 1 tbsp
------------------ | ------------: | ------------:
flour              | 140g          |
white/brown sugar  | 200g
powdered sugar     | 120g
cocoa powder       |  85g
butter<sup>1</sup> | 225g          | 15g 

<sup>1</sup> 1 cup butter = 8 ounces = 2 sticks

## Oven Temperature

|        |  Fahrenheit | Celsius
-------- | ----------: | ------------:
cool     | 200°F | 90°C
slow     | 300°F | 150°C
moderate | 350°F | 180°C
hot      | 400°F | 200°C
fast     | 450°F | 230°C
```
Heh -- "multi-line suggestion" for all lines!  This seems easier than doing another PR.  You can preview the result here: https://github.com/RyanSkraba/cookbook/blob/patch-1/us_to_metric.md

Reply:
I think the only non-formatting change was dropping some information about the half cup of butter...